lnwire/accept_channel.go

```diff
@@ -86,6 +86,11 @@ type AcceptChannel struct {
 	// base point in order to derive the revocation keys that are placed
 	// within the commitment transaction of the sender.
 	FirstCommitmentPoint *btcec.PublicKey
+
+	// UpfrontShutdownScript is the script to which the channel funds should
+	// be paid when mutually closing the channel.  This is to be enforced by
+	// the remote node even if this node is compromised later.
+	UpfrontShutdownScript DeliveryAddress
```

Comment:
reuse the existing `PkScript` type? https://github.com/lightningnetwork/lnd/blob/master/lnwire/lnwire.go#L27

Reply:
scratch this, i thought `DeliveryAddress` was added in this pr